src/lib.rs

```diff
@@ -43,18 +46,83 @@ pub struct UPClientZenoh {
     query_map: Arc<Mutex<HashMap<String, Query>>>,
     // Save the callback for RPC response
     rpc_callback_map: Arc<Mutex<HashMap<String, Arc<UtransportListener>>>>,
+    // Used to identify different callback
     callback_counter: AtomicU64,
+    // Source UUri in RPC
+    uuri: UUri,
```

Comment:
IMHO let's name it `rpc_request_source_uuri` or something similarly descriptive, rather than that be in the comment.

WDYT?

Reply:
Since the uuri is used in both notification and RPC request, maybe the name should be something like `source_uuri`
Naming is always difficult for me 😅

Reply:
Yup, makes sense now that I understand it's going to be used for both Notification and Request source UUris :slightly_smiling_face:

Reply:
Done.